protocol_driver_grpc.cc

```diff
@@ -199,8 +199,7 @@ void GrpcInlineServerDriver::HandleConnectFailure(
     std::string_view local_connection_info) {}
 
 void GrpcInlineServerDriver::ShutdownServer() {
-  if (server_)
```

Comment:
Probably should put this in a separate cosmetic change pull request, or just revert it.